sdk/keyvault/keyvault-certificates/src/index.ts

```diff
@@ -239,12 +238,21 @@ function toCorePolicy(
 
   return {
     id,
-    lifetimeActions: policy.lifetimeActions,
+    lifetimeActions: policy.lifetimeActions
+      ? policy.lifetimeActions.map((action) => ({
+          action: { actionType: action.action },
```

Comment:
Especially now that you've removed the interface I mentioned before that shouldn't have an `actionType` member, shouldn't this line just be `action: action.action`?

Reply:
We chatted a bit off-line. This is actually part of the repack for the core service, so this is correct.